src/ring_theory/localization.lean

```diff
@@ -379,6 +381,13 @@ variables (M)
 lemma mk'_surjective (z : S) : ∃ x (y : M), mk' S x y = z :=
 let ⟨r, hr⟩ := is_localization.surj _ z in ⟨r.1, r.2, (eq_mk'_iff_mul_eq.2 hr).symm⟩
 
+/-- The localization of a `fintype` is a `fintype`. Cannot be an instance. -/
+noncomputable def fintype' {R : Type u} [comm_ring R] [fintype R] (S : submonoid R) (L : Type v)
+  [comm_ring L] [algebra R L] [is_localization S L] : fintype L :=
+have _ := classical.prop_decidable, by exactI
```

Comment:
Can you do this with a `decidable` argument instead? Normally fintype instances should be computable unless it's unavoidable.

Reply:
I think `is_localization.mk'` is noncomp anyways, sadly

Reply:
That's a fair argument. I guess the problem is `localization_map`, which could be adjusted to hold a direct map to the units. Obviously out of scope here.

Reply:
@eric-wieser So what do you suggest? Merge as is?